src/audio/copier.c

```diff
@@ -183,9 +183,6 @@ static struct comp_dev *create_host(struct comp_dev *parent_dev, struct copier_d
 	if (!dev)
 		return NULL;
 
-	list_init(&dev->bsource_list);
-	list_init(&dev->bsink_list);
-
```

Comment:
are you sure this one and the one below are correct? `create_host()` and `create_dai()` call `drv->ops.create()` manually and `.create()` now doesn't initialise lists, so we initialise them here. `create_host()` and `create_dai()` are indeed called from a `.new()` method but that's creating a different device object.